lib/api/v3/utilities/path_helper.rb

```diff
@@ -46,6 +46,14 @@ def self.attachment(id)
             "#{root}/attachments/#{id}"
           end
 
+          def self.attachment_download(id)
+            "#{root_path}attachments/#{id}"
+          end
```

Comment:
Why not use the Rails path inside this method here so that you avoid the repetition but have a consistent path registry for the whole of the API.